.github/CONTRIBUTING.md

```diff
@@ -18,9 +18,10 @@ the preferred channel for [bug reports](#bug-reports), [features requests](#feat
 and [submitting pull requests](#pull-requests), but please respect the following
 restrictions:
 
-* Please **do not** use the issue tracker for personal support requests.  Stack
+* Please **do not** use the issue tracker for personal support requests. Stack
   Overflow ([`bootstrap-5`](https://stackoverflow.com/questions/tagged/bootstrap-5) tag),
-  [Slack](https://bootstrap-slack.herokuapp.com/) or [IRC](/README.md#community) are better places to get help.
+  [Slack](https://bootstrap-slack.herokuapp.com/) or [IRC](/README.md#community) are better places to get help for things related to Bootstrap.
+  For Boosted specific personal support requests please contact directly Boosted Core Team via Orange internal tools.
```

Comment:
I would had a comma between "requests" and "please" for a better readability of the sentence.

Reply:
I hesitated so I got my answer 😄 Modified via https://github.com/Orange-OpenSource/Orange-Boosted-Bootstrap/pull/1223/commits/26a37f40da18a3b49b20ecbc53840a2d70096ffe